### PR TITLE
Added a ChatGPT 3.5 Jailbreak I Have Created

### PR DESCRIPTION
**Pull request by MarkCyber**
This is a chatgpt jailbreak for chatgpt 3.5 I have created that utilizes multiple steps. This jailbreak utilizes hexadecimal to bypass restrictions and allow chatgpt to write malware / hacking scripts that it would not normally do.

Proof of concept: 
ProofOfConcept

- [yes ] I've confirmed the prompt works well
- [yes ] I've added `Contributed by: [@yourusername](https://github.com/yourusername)`
- [yes ] I've added to the README.md
- [yes ] I've added to the `prompts.csv`
- [yes ] Escaped quotes by double-quoting them
- [yes ] No spaces after commas after double quotes. e.g. `"Hello","hi"`, not `"Hello", "hi"`
- [yes ] Removed "Act as" from the title on CSV

Please make sure you've completed all the checklist.

### CODE REVIEW

**Review by lozanopo — APPROVED** [review]

**Comment by MarkCyber** [comment]
What is the process to have this merged after getting it approved? Is there anything else I need to do on my end?

**Review by lozanopo — APPROVED** [review]

**Review by lozanopo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,100 +1,37 @@
-<p align="center"><h1>🧠 Awesome ChatGPT Prompts</h1></p>
```
Comment (davidsneighbour) [inline README.md:1]: Deleting these lines, down to line 97 is not connected to your prompt.

File: README.md [hunk 2]
```diff
@@ -1,100 +1,37 @@
...
-### Want to Learn How to write image prompts for Midjourney AI?
-
-I've authored an e-book called **"The Art of Midjourney AI: A Guide to Creating Images from Text"**.
+# Prompts
 
-📖 **[Read the e-book](https://fka.gumroad.com/l/the-art-of-midjourney-ai-guide-to-creating-images-from-text)**
+## ChatGPT (3.5) The Hacker Jailbreak
+Contributed by: [MarkCyber](https://github.com/MarkCyber)
 
----
```
Comment (davidsneighbour) [inline README.md:71]: I doubt that intention is a factor in allowing coarse language in a public document. There is a button "collapsible content" in the editor that adds some Github-compatible markdown to collapse text which you might want to use for that?

**Review by davidsneighbour — CHANGES_REQUESTED** [review]

**Comment by MarkCyber** [comment]
Okay, I did not do this intentionally. I will fix this now. Thank you for letting me know!

**Comment by MarkCyber** [comment]
Started a new pull request with all of these changes fixed

**Pull request closed without merging**